src/CompletionHandler.php

```diff
@@ -34,17 +34,23 @@ class CompletionHandler
      */
     protected $helpers = array();
 
+    /**
+     * @var bool
+     */
+    protected $commandsHidable;
+
     public function __construct(Application $application, CompletionContext $context = null)
     {
         $this->application = $application;
         $this->context = $context;
+        $this->commandsHidable = method_exists('\Symfony\Component\Console\Command\Command', 'isHidden');
```

Comment:
This is only used in one place, so either inline it or put it in a protected method